registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -238,6 +239,7 @@
     "sha256:e4fd47e1786e6a3943a55e3a776c7d58fe9cb4b3870c71dc76e8bfa29b20b63c": ["v1.11.0-alpha.0"]
     "sha256:6e78e8f2ebb0f5a7ac9ca77ab1005313cd822cedec5e7b149a8c8c779943c07a": ["v1.11.0-alpha.1"]
     "sha256:b4b5bdf682c1badf8bffe422f99280bca6775fd5ae0e2cc299bad494f5ca2853": ["v1.11.0-alpha.2"]
+    "sha256:3f791c0c0ead3c788723030563f1ae4ce9eae7a88cff9f9293ef81b1e3a21dc1": ["v1.11.0-beta.0"]
```

Comment:
SHA `3f791c0c0ead ` matches clusterctl v1.11.0-beta.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:3f791c0c0ead3c788723030563f1ae4ce9eae7a88cff9f9293ef81b1e3a21dc1/details?inv=1&invt=Ab21Sw). 👍🏻